frog/imports/client/Wiki/index.js

```diff
@@ -74,7 +80,8 @@ type WikiCompStateT = {
 };
 
 class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
-  wikiId: string = this.props.match.params.wikiId;
+  wikiId: string =
```

Comment:
We are setting a prop value as a class global. This will break React rerendering on prop change. If the props change, this value will not get changed, causing a potential bug.

Reply:
Fair point, I faced a bug due to this one!

Reply:
There are no immediate bugs due to this issue, though this could be addressed later in a refactor of the Wiki.